scripts/core/LEVEL_ZERO.rst

```diff
@@ -139,8 +139,15 @@ Environment Variables
 | UR_L0_DISABLE_USM_ALLOCATOR                 | Controls the use of the USM allocator.                       | "0": USM allocator is enabled.                               | "0"              |
 |                                             |                                                              | Any other value: USM allocator is disabled.                  |                  |
 +---------------------------------------------+--------------------------------------------------------------+--------------------------------------------------------------+------------------+
-
+| UR_L0_CMD_BUFFER_USE_IMMEDIATE_APPEND_PATH  | Controls which command-buffer implementation path is used.   | "1": the immediate append path will always be enabled as     | Unset            |
```

Comment:
Which path is tested will depend on the CI runner. We should more explicit about it by testing twice - with and without the "immediate append path" enabled.

Reply:
At the moment, the CI drivers are all older than 30898 and I would assume that that will be the case for the foreseeable future. So it will always use the old path. 

I think that, once this extension is not experimental anymore, if we are still using 2 paths in the code, we could add more testing to CI. But for now, I don't think there is much point in doing so.

Reply:
AFAIK our PVC runners have a new enough driver (the newest available rolling is `24.39.31294.21`, but fair, we can expand CI later. Please create an issue.

Reply:
Created ticket https://github.com/oneapi-src/unified-runtime/issues/2373